bears/python/PEP8Bear.py

```diff
@@ -41,14 +42,69 @@ def run(self, filename, file,
                    'max_line_length': max_line_length,
                    'indent_size': indent_size}
 
+        errors = self.list_pep8_errors(source=file, options=options)
```

Comment:
If I understand this correctly, you run `pycodestyle` independently and then match it with result from `autopep8`?

I think it will be unreliable because the coala-generated `diffs` below is not guaranteed to match 1-on-1 with list from `pycodestyle`.